src/pharmpy/plugins/nonmem/model.py

```diff
@@ -85,13 +86,12 @@ def convert_model(model):
     nm_model.value_type = model.value_type
     nm_model._data_frame = model.dataset
     nm_model._estimation_steps = model.estimation_steps
-    nm_model._datainfo = model.datainfo
     nm_model._initial_individual_estimates = model.initial_individual_estimates
     nm_model.observation_transformation = model.observation_transformation.subs(
         model.dependent_variable, nm_model.dependent_variable
     )
     nm_model.description = model.description
-    nm_model.update_source()
+    nm_model.update_source(nofiles=True)
```

Comment:
This skips an unsuccessful file overwrite attempt that occurs when the dataset is updated (for instance when we remove the RATE column from the dataframe). This is not needed if we make sure the RATE column does not get deleted.